client/src/modules/notification/components/Notification.jsx

```diff
@@ -4,9 +4,13 @@ import { FormattedMessage } from 'react-intl';
 
 import MessageBox from 'common/components/MessageBox';
 import { MessageType as NotificationType } from 'common/constants/enums';
+import { mapObject } from 'common/utils/helpers';
```

Comment:
why not lodash _map ??
https://lodash.com/docs/4.17.15#map

Reply:
Because lodash's _map returns an array. We need an object. This helper (we use it on backend already) returns an object with all values modyfied by callback you pass as an argument